src/coverage.ts

```diff
@@ -32,7 +32,7 @@ export async function handleCoverageInfoFiles(run: vscode.TestRun, coverageInfoF
         }
         const sections = await lcovParser({ from: contents });
         for (const section of sections) {
-            const coverage = new vscode.FileCoverage(vscode.Uri.file(section.path),
+            const coverage = new vscode.FileCoverage(vscode.Uri.file(section.path.trim()),
```

Comment:
Do we also need to normalize the paths based on the os? 

For example, with our normalize methods in the utils file?

Reply:
I'm honestly not sure. I'll let you decide if that makes the most sense. I just tried changing the line to `const coverage = new vscode.FileCoverage(vscode.Uri.file(platformNormalizePath(section.path.trim())),` (after also importing that function) and that still works. The normalize function seemingly does not remove the "trailing" carriage return, so the `trim()` is still needed.

Reply:
Sorry for the delay in response, I think it's likely safer to go ahead and normalize, so if you could update with the `platformNormalizePath`, that would be great and we can approve and merge this PR. 

Thanks for your patience.

Reply:
Done, and no worries at all.